beacon_node/eth2_libp2p/src/peer_manager/mod.rs

```diff
@@ -940,19 +946,33 @@ impl<TSpec: EthSpec> PeerManager<TSpec> {
         let connected_peer_count = self.network_globals.connected_peers();
         if connected_peer_count > self.target_peers {
             //remove excess peers with the worst scores, but keep subnet peers
-            for (peer_id, _) in self
+            //must also ensure that the outbound-only peer count does not go below the minimum threshold
```

Comment:
We should have mentioned this earlier, sorry about that. We try to maintain a comment style of Uppercase first letter, space after `//`and final point. It's a bit recent that we formalized this [here](https://lighthouse-book.sigmaprime.io/contributing.html#comments) which is probably why the comment on the previous line to this one does not fit the desired style either. Would you mind updating that one too?